libs/quasselc/patches/001-respect-cflags-ldflags.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/Makefile b/Makefile
+index 7994eea..b1f8d83 100644
+--- a/Makefile
++++ b/Makefile
+@@ -2,11 +2,11 @@ prefix ?= /usr/local
+ libdir ?= $(prefix)/lib
+ includedir ?= $(prefix)/include
+ 
+-CFLAGS:=-Wall -g -Wextra $(shell pkg-config glib-2.0 --cflags) -Wswitch-enum -std=gnu11 -O2 -fPIC
++CFLAGS+=-Wall -g -Wextra $(shell pkg-config glib-2.0 --cflags) -Wswitch-enum -std=gnu11 -O2 -fPIC
```

Comment:
The build system tend to have `-Os` in cflags and according to manpage of gcc `If you use multiple -O options, with or without level numbers, the last such option is the one that is effective.`  So in this case the build system will lose its control of how this package will be built.  I may suggest just dropping that `-O2` option altogether.